lib/dvbjs.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 var request = require('request');
```

Comment:
Fixed in https://github.com/kiliankoe/dvbjs/commit/fb96b9480c3422afdfce7290da16e3630fbb419f